packages/dao/contracts/Api3BaseTemplate.sol

```diff
@@ -85,15 +84,6 @@ contract Api3BaseTemplate is APMNamehash, IsContract {
 
     /* ACL */
 
-    function _createPermissions(ACL _acl, address[] memory _grantees, address _app, bytes32 _permission, address _manager) internal {
```

Comment:
Are you sure that we don't need all that functionality anymore? Btw, as I understand that was partially a reason for 2 voting apps instead of 1

Reply:
I was following recommendations from the first audit, we can return the functionality if needed later on